packages/office-ui-fabric-react/src/components/DetailsList/examples/DetailsList.CustomGroupHeaders.Example.tsx

```diff
@@ -110,4 +104,16 @@ export class DetailsListCustomGroupHeadersExample extends React.Component<{}, {}
       props!.onToggleCollapse!(props!.group!);
     };
   }
+
+  /** alterColumnNames modifies columns from example data to clarify their usage in examples.
+   * It is not necessary to replicate this function in your own code.
+   */
+  private _alterColumnNames = (cols: IColumn[]): IColumn[] => {
+    cols.forEach(c => {
+      if (c.name === 'thumbnail') {
```

Comment:
Can't you just change the column name in the example data instead? (then fix anything that breaks)

Alternatively you could either fix the thumbnail column to have a proper renderer, or add code to hide it in most circumstances.

Reply:
I explored providing a consistent renderer for thumbnails across our examples, but it changes our more tabular `DetailsList` examples too much.

![image](https://user-images.githubusercontent.com/706967/57662424-543ed100-75a4-11e9-8aca-f8292c13721e.png)

We could make the thumbnails smaller, but at that point they would be comically small and unrealistic.

After noticing the above, I'm of the opinion that we minimize the cases were we showcase thumbnail rendering in our `DetailsList` examples.

Reply:
Hmm okay, thanks for investigating. Similarly to what JD said, I'd be in favor of removing the column entirely (or *maybe* adding an option to include it, which is false by default).